hfst-optimized-lookup.cc

```diff
@@ -253,6 +253,7 @@ int main(int argc, char **argv)
 
         case 'e':
           echoInputsFlag = true;
+          break;
```

Comment:
Is this bug in upstream? Not that you need to give them a PR, I'm just curious.

Reply:
Good call: https://github.com/hfst/hfst/pull/534